src/components/Chat/ActiveChatsComponent.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useMemo, useState } from 'react';
+import { IChat } from '../../interfaces';
+
+interface IProps {
+  availableChats: IChat[];
+  showMessages(index: number): void;
+}
+
+const ActiveChatsComponent = (props: IProps) => {
+  const [activeChat, setActiveChat] = useState<number>(0);
+
+  const onClickHandler = (index: number): void => {
+    props.showMessages(index);
+    setActiveChat(index);
+  };
+
+  const activeChatList = () =>
+    props.availableChats.map((chat, index) => (
+      <div
+        onClick={() => onClickHandler(index)}
+        key={index}
```

Comment:
Hvis du kan unngå det så bør du egentlig ikke bruke index som key (og det er uansett default). Se [her](https://reactjs.org/docs/lists-and-keys.html#keys) for hvorfor.